tests/Functional/BundleInitializationTest.php

```diff
@@ -12,10 +12,11 @@
 namespace Translation\PlatformAdapter\Loco\Tests\Functional;
 
 use Http\HttplugBundle\HttplugBundle;
+use Nyholm\BundleTest\TestKernel;
 use Translation\PlatformAdapter\Loco\Bridge\Symfony\TranslationAdapterLocoBundle;
 use Translation\PlatformAdapter\Loco\Loco;
 
-class BundleInitializationTest
+class BundleInitializationTest extends TestKernel
```

Comment:
TestKernel is an instance of `Symfony\Component\HttpKernel\Kernel` : https://github.com/SymfonyTest/symfony-bundle-test/blob/master/src/TestKernel.php and it's not an instance of PHPUnit TestCase, so the this test will not work (will probably not even be executed by PHPUnit)

And `$this->createKernel()` used below still does not exist

Reply:
This is the issue with the last tag available: https://github.com/SymfonyTest/symfony-bundle-test/issues/65

Reply:
I guess you wanted to answer to the other thread about the version number.

But this test needs to be fixed anyway

Reply:
I just fixed the test now